inclusive-design/idrc
# feat: add news page (resolves #11)

## Description

Adds a News page.

## Steps to test

1. Visit News page, check out the two linked news items.

**Expected behavior:** Matches designs.

## Additional information

Pagination is implemented but not yet testable because we have less than six news items.

## Related issues

- Resolves #11.
